apps/media_lights_sync/media_lights_sync.py

```diff
@@ -21,6 +22,7 @@ def initialize(self):
         args = self.args
         self.lights = args["lights"]
         self.ha_url = args.get("ha_url", None)
+        self.verify_cert = args.get("verify_cert", True)
```

Comment:
We should add that option in the README https://github.com/ericmatte/ad-media-lights-sync#app-configuration